app/src/main/res/values/dimens.xml

```diff
@@ -44,4 +44,19 @@
     <dimen name="dimens_168dp">168dp</dimen>
     <dimen name="dimens_200dp">200dp</dimen>
 
+    <dimen name="spacing_xsmall">2dp</dimen>
```

Comment:
Please try to follow the exisiting naming convention (_xdp suffix.)and create a new issue if you suggest us to follow a better naming convention for dimensions. Add only dimensions that are needed. If possible, we will try to remove all the odd dimensions (3dp, 5dp,..) and use even dimensions (multiple of 4 in our code).

Reply:
Okay , I have corrected those strings accordingly.